openbsd/OpenBSDMachine.h

```diff
@@ -45,8 +45,8 @@ typedef struct OpenBSDMachine_ {
 
    long fscale;
    int cpuSpeed;
-   int pageSize;
-   int pageSizeKB;
+   size_t pageSize;
```

Comment:
Not sure if making them `size_t` is the correct fix here.

Reply:
That's literally what size_t is for. ;-) Denoting the number of elements that can fit into memory.